src/org/traccar/reports/ReportUtils.java

```diff
@@ -33,23 +33,23 @@ private ReportUtils() {
 
     public static String getDistanceUnit(long userId) {
         String unit = Context.getPermissionsManager().getUser(userId).getDistanceUnit();
-        if (unit == null) {
+        if (unit == null || Context.getPermissionsManager().getServer().getForceSettings()) {
```

Comment:
There is actually problem with this code. If server value is not set, we still want to use user value. Web app handles this situation.